docs/tutorials/quick_start_rake_tasks.md

```diff
@@ -0,0 +1,205 @@
+      nfs_server: none
+      consoleport: 443
+      pooling_api: http://vmpooler.delivery.puppetlabs.net/
+    
+
+
+Vagrant file (Ubuntu 14 master and agent):
+
+    ---
+    HOSTS:
+      ubuntu1404-64-1:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - master
+        - database
+        - dashboard
+        - classifier
+        - default
+      ubuntu1404-64-2:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - frictionless
+    CONFIG:
+      nfs_server: none
+      consoleport: 443
+      box_url: https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm
+      box: puppetlabs/ubuntu-14.04-64-nocm
+
+
+For more info on host generation and what these configs represent see:
+
+* [Creating A Test Environment](creating_a_test_environment.md)
+
+
+### gen_pre_suite
+
+To run:
+
+    rake beaker_quickstart:gen_pre_suite
+    
+This task will generate a default Beaker pre-suite file.
+
+The gen_pre-suite task will create a file 'default_pre_suite.rb' in acceptance/setup.
+
+If the file already exists, it will not be overwritten. This will allow you to edit the pre_suite file yourself if required.
+
+pre_suite file:
+
+    install_puppet
+  
+See the other tutorial docs in this directory for more information on pre-suites:
+
+* [Test Suites](test_suites.md)
+
+
+### gen_smoke_test
+
+To run:
+
+    rake beaker_quickstart:gen_smoke_test
+    
+This task will generate a default Beaker smoke test file.
+
+The gen_smoke_test task will create a file 'default_smoke_test.rb' in acceptance/tests.
+
+If the file already exists, it will not be overwritten. This will allow you to edit the test file yourself if required.
+
+smoke test file:
+
+    test_name 'puppet install smoketest'
+    step 'puppet install smoketest: verify \'puppet help\' can be successfully called on
+    all hosts'
+        hosts.each do |host|
+          on host, puppet('help')
+        end
+  
+This smoke test will check that Puppet has been successfully installed on the hosts.
+
+For more information on the Beaker dsl methods available to you in your tests see:
+
+- [Beaker dsl](how_to/the_beaker_dsl.md)
+
+
+### run_test
+
+To run:
+  
+    rake beaker_quickstart:run_test
+    
+This task will run the above 3 tasks in sequential order and then execute a Beaker test run using all 3 files.
+
+    beaker --hosts acceptance/config/default_vmpooler_hosts.yaml --pre-suite acceptance/setup/default_pre_suite.rb --tests 
+    acceptance/tests/default_smoke_test.rb
+
+You will end up with provisioned hosts, with puppet installed and a test check executed to make sure that puppet was indeed installed.
+
+For more information on running Beaker tests see:
+
+* [Test run](test_run.md)

```

Comment:
same as above: if we're not going to have a list of multiple references, I think it would be better to include the link in the sentence itself.